source/agora/consensus/protocol/Nominator.d

```diff
@@ -280,7 +280,7 @@ extern(D):
     public void setQuorumConfig (ref const(NodeID) node_id,
         const(QuorumConfig)[NodeID] quorums) nothrow @safe
     {
-        assert(!this.is_nominating);
+        assert(!this.is_nominating || this.nomination_timer is null);
```

Comment:
> On a block being externalized, the timer for a nomination process stops, and the newly constructed quorums are set.

I couldn't find the timer stopping when block is externalized. Nomination round gets stopped but it doesn't stop the actual timer.

Reply:
Right. It's embarrassing. I checked the wrong target, `nomination_timer` to see if the nomination round gets stopped. And I need to have another approach also.
Will modify it.
Thanks.